tests/test_sensor_download.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+
+from tests import test_authorization as Authorization
+from falconpy import sensor_download as FalconSensorDownload
+
+sys.path.append(os.path.abspath('src'))
+AllowedResponses = [200, 429]  # Adding rate-limiting as an allowed response for now
+appId = "pytest-sensor_download-unit-test"
+auth = Authorization.TestAuthorization()
+auth.serviceAuth()
+
+sensor_download_client = FalconSensorDownload.Sensor_Download(access_token=auth.token)
+
+class TestSensorDownload():
+    
+    @staticmethod
+    def _get_cid():
+        resp = sensor_download_client.GetSensorInstallersCCIDByQuery()
+        return True if resp["status_code"] in AllowedResponses else False
+
+    @staticmethod
+    def _get_multiple_shas():
+        params = {"filter": 'platform:"windows"', "sort": "release_date|desc"}
+        shas = sensor_download_client.GetSensorInstallersByQuery(params)["body"].get("resources", ["b7dbe32495c6e23ca2e04784b98d5a1770163b9f42524cafc062292b533f03c9"])
+        return shas
+
+    def _download_sensor(self):
+        file_name = "falconwinddows.exe"
+        directory_path = "sensor_downloads"
+        sha_id = self._get_multiple_shas()[0]
+        resp = sensor_download_client.DownloadSensorInstallerById(_id=sha_id, file_name=file_name, download_path=directory_path)
+        return resp
```

Comment:
This unit test is a little strange but I think I understand how it works. From what I'm reading here we're returning the value of **resp** which in this case would contain the binary for the download.  Python would calculate that to equal a _True_ that the assert statement is expecting down below.  (Or am I misreading?)